aten/src/ATen/native/cuda/CUDALoops.cuh

```diff
@@ -42,6 +42,8 @@
 #include <c10/util/TypeCast.h>
 #include <c10/util/C++17.h>
 
+#include <thrust/tuple.h>
```

Comment:
All your changes in this file should go to Loops.cuh instead of CUDALoops.cuh, because we will need ROCm support as well, and this file is not included on ROCm.